easybuild/easyconfigs/l/libX11/libX11-1.6.3-foss-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'libX11'
+version = '1.6.3'
+versionsuffix = '-Python-2.7.10'
```

Comment:
@boegel - yes, I need to remove these extra files from this PR. They're in a separate PR now, anyway and are irrelevant here, where I'm concentrating on `foss-2016a` now. Will do so later today.